### PR TITLE
updated karate tunes

### PR DESCRIPTION
**Pull request by sugaarK**
updated karate 6s tune to slider

and added suagrK rates as a file to all karate tunes
along with the RClink files

### CODE REVIEW

File: presets/4.4/tune/karate/karate_brushless_whoop.txt [hunk 1]
```diff
@@ -121,26 +121,62 @@ set simplified_pitch_pi_gain = 95
...
+
 #$ OPTION_GROUP END
 
 #$ OPTION_GROUP BEGIN: Optional rates
-
-    #$ OPTION BEGIN (UNCHECKED): sugarK's whoop rates
+    #$ OPTION BEGIN (UNCHECKED): sugarK's rates
+        #$ INCLUDE: presets/4.3/rates/SugarK.txt
         set thr_expo = 80
         set roll_expo = 54
```
Comment (limonspb) [inline presets/4.4/tune/karate/karate_brushless_whoop.txt:178]: this could be wrong

**Review by limonspb — CHANGES_REQUESTED** [review]
And lets make karate race as experimental temporarly cuz it has lots of text changes.

File: presets/4.4/tune/karate/karate_brushless_whoop.txt [hunk 2]
```diff
@@ -121,26 +121,62 @@ set simplified_pitch_pi_gain = 95
...
+
+    #$ OPTION END
+    
+    #$ OPTION BEGIN (UNCHECKED): ELRS 1000Hz
+        # ELRS 1000Hz
+       
+        feature RX_SERIAL
+        set serialrx_provider = CRSF
+
+         #$ INCLUDE: presets/4.3/rc_link/generic/1000hz_race.txt
```
Comment (KarateBrot) [inline presets/4.4/tune/karate/karate_brushless_whoop.txt:149]:
```suggestion
        #$ INCLUDE: presets/4.3/rc_link/generic/1000hz_race.txt
```

File: presets/4.4/tune/karate/karate_brushless_whoop.txt [hunk 3]
```diff
@@ -121,26 +121,62 @@ set simplified_pitch_pi_gain = 95
...
+         
+    #$ OPTION END
+    
+    #$ OPTION BEGIN (UNCHECKED): Ghost 250Hz
+        # Ghost 250Hz
+      
+        feature RX_SERIAL
+        set serialrx_provider = GHST
+
+         #$ INCLUDE:presets/4.3/rc_link/generic/250hz_race.txt
```
Comment (KarateBrot) [inline presets/4.4/tune/karate/karate_brushless_whoop.txt:159]:
```suggestion
        #$ INCLUDE:presets/4.3/rc_link/generic/250hz_race.txt
```

File: presets/4.4/tune/karate/karate_brushless_whoop.txt [hunk 4]
```diff
@@ -121,26 +121,62 @@ set simplified_pitch_pi_gain = 95
...
+
     #$ OPTION END
+
+    #$ OPTION BEGIN (UNCHECKED): Ghost 500Hz
+        # Ghost 500Hz
+       
+        feature RX_SERIAL
+        set serialrx_provider = GHST
+
+         #$ INCLUDE: presets/4.3/rc_link/generic/500hz_race.txt
```
Comment (KarateBrot) [inline presets/4.4/tune/karate/karate_brushless_whoop.txt:169]:
```suggestion
        #$ INCLUDE: presets/4.3/rc_link/generic/500hz_race.txt
```

**Review by KarateBrot — DISMISSED** [review]

File: presets/4.4/tune/karate/karate_brushless_whoop.txt [hunk 5]
```diff
@@ -121,26 +121,63 @@ set simplified_pitch_pi_gain = 95
...
+
+    #$ OPTION END
+    
+    #$ OPTION BEGIN (UNCHECKED): ELRS 1000Hz
+        # ELRS 1000Hz
+       
+        feature RX_SERIAL
+        set serialrx_provider = CRSF
+
+         #$ INCLUDE: presets/4.3/rc_link/generic/1000hz_race.txt
```
Comment (KarateBrot) [inline presets/4.4/tune/karate/karate_brushless_whoop.txt:149]:
```suggestion
        #$ INCLUDE: presets/4.3/rc_link/generic/1000hz_race.txt
```

File: presets/4.4/tune/karate/karate_brushless_whoop.txt [hunk 6]
```diff
@@ -121,26 +121,63 @@ set simplified_pitch_pi_gain = 95
...
+         
     #$ OPTION END
+    
+    #$ OPTION BEGIN (UNCHECKED): Ghost 250Hz
+        # Ghost 250Hz
+      
+        feature RX_SERIAL
+        set serialrx_provider = GHST
+
+         #$ INCLUDE:presets/4.3/rc_link/generic/250hz_race.txt
```
Comment (KarateBrot) [inline presets/4.4/tune/karate/karate_brushless_whoop.txt:159]:
```suggestion
        #$ INCLUDE:presets/4.3/rc_link/generic/250hz_race.txt
```

File: presets/4.4/tune/karate/karate_brushless_whoop.txt [hunk 7]
```diff
@@ -121,26 +121,63 @@ set simplified_pitch_pi_gain = 95
...
+
+    #$ OPTION END
+
+    #$ OPTION BEGIN (UNCHECKED): Ghost 500Hz
+        # Ghost 500Hz
+       
+        feature RX_SERIAL
+        set serialrx_provider = GHST
+
+         #$ INCLUDE: presets/4.3/rc_link/generic/500hz_race.txt
```
Comment (KarateBrot) [inline presets/4.4/tune/karate/karate_brushless_whoop.txt:169]:
```suggestion
        #$ INCLUDE: presets/4.3/rc_link/generic/500hz_race.txt
```

**Review by KarateBrot — CHANGES_REQUESTED** [review]
You reverted the changes by accident

**Review by KarateBrot — APPROVED** [review]

**Pull request merged**